backend/src/packages/files/aws.service.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  GetObjectCommand,
+  PutObjectCommand,
+  type PutObjectCommandInput,
+  S3Client,
+  type S3ClientConfig,
+} from '@aws-sdk/client-s3';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+
+import { type AWSUploadRequestDto } from './libs/types/types.js';
+
+type AWSServiceDependencies = {
+  region: string;
+  accessKeyId: string;
+  secretAccessKey: string;
+  bucketName: string;
+};
+
+class AWSService {
```

Comment:
AWS is not consist of only S3)

Reply:
should I rename the service to S3 then?

Reply:
I think you can name it file service

Reply:
@fshabanov but I already have file.service, is there another name I should go with?

Reply:
let's name it `S3` and move to libs/packages/s3/s3.package.ts and then re-export from libs/packages/file/file.ts